### ISSUE TITLE
NVDA sometimes stops processing all UIA events until it is restarted

### ISSUE THREAD

**Issue by Neurrone** [nvaccess/nvda#11423]
### Steps to reproduce:

1. Do something such as causing large amounts of text to be written to the console. Its the most reliable way I've gotten this to happen so far, but by far not the only way.

### Actual behavior:

In the start menu, try typing. Alt tab to the command prompt and attempt to use the review cursor to review text. Nothing happens.

### Expected behavior:

Both things should work.

#### NVDA installed/portable/running from source:
Installed

#### NVDA version:
alpha-20581,8d5d1802

#### Windows version:
Windows 10 2004

#### Other information about your system:

* The selective UIA registration setting was not enabled
* Use UIA in the command prompt is enabled

Something else that might be related to this happens with Unigram. I realized that the first time that Unigram starts when you first restart a machine, NVDA always fails to read anything when arrowing about. It requires you to restart Unigram for it to read normally.

#### Does the issue still occur after restarting your computer?
Yes

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
No

#### If addons are disabled, is your problem still occuring?
Unknown

#### Did you try to run the COM registry fixing tool in NVDA menu / tools?
No

**Comment by leonardder** [comment]
What happens if selective UIA registration is on? Do you also suffer from this?

**Comment by Neurrone** [comment]
Will try it and see, haven't experimented with turning on the setting yet.

**Comment by Adriani90** [comment]
@Neurrone could you please give an update? Thanks.

**Comment by Neurrone** [comment]
Have enabled the option, but I'll need to leave it on and test for several weeks before knowing for sure.

**Comment by codeofdusk** [comment]
Duplicate of #11002.

**Comment by Neurrone** [comment]
Closing in favour of #11002.
